airflow/models/dagrun.py

```diff
@@ -388,55 +389,88 @@ def active_runs_of_dags(
         return dict(iter(session.execute(query)))
 
     @classmethod
-    def next_dagruns_to_examine(
-        cls,
-        state: DagRunState,
-        session: Session,
-        max_number: int | None = None,
-    ) -> Query:
+    @retry_db_transaction
+    def running_dag_runs_to_examine(cls, session: Session) -> Query:
```

Comment:
Small nit/question. Should we name it `get_running_dag_runs_to_examine` to be explicit that we only get these dag runs? That applies for the other function as well

Reply:
done